ui/app/mainui/activitycenter/views/ActivityNotificationContactVerification.qml

```diff
@@ -0,0 +1,32 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+import shared 1.0
+import shared.panels 1.0
+import utils 1.0
+
+import "../panels"
+
+ActivityNotificationMessage {
+    id: root
+
+    messageSubheaderComponent: StatusBaseText {
+        text:  qsTr("Identity Verification Question:")
+        color: Theme.palette.baseColor1
```

Comment:
No need to fuzz around with text color :)

Reply:
By default it's black, but i need gray:
<img width="549" alt="Screenshot 2022-11-21 at 17 39 44" src="https://user-images.githubusercontent.com/2522130/203298934-c84c12e6-89e4-40e8-9e1c-41ecd1dd3741.png">